assets/sass/vendor/_mdc-dialog.scss

```diff
@@ -24,7 +24,8 @@
 	}
 
 	.mdc-dialog__actions {
-		justify-content: flex-start;
+		justify-content: space-between;
+		margin: 2em 0;
```

Comment:
The space _below_ the buttons here looks odd to me. I know the IB said `2em` but it seems like quite a lot, especially below the button. What does `1em` look like?

https://github.com/google/site-kit-wp/pull/1458/files#diff-abeb2930ae311ff937d08ecc500f6f3c to me is a bit too much whitespace.

Reply:
Yeah @tofumatt 1em looks better to me, should I go ahead and commit?
![modal-dialog-1em](https://user-images.githubusercontent.com/4214099/80537304-a9f8be80-8958-11ea-9d49-be7f8894ff96.png)

Reply:
Looks good, :shipit:

Reply:
Great, updated and pushed. 👍